sdk-api.yaml

```diff
@@ -890,6 +963,25 @@ components:
                       nullable: true
                       additionalProperties:
                         type: string
+        planOverrides:
+          type: object
+          nullable: true
+          properties:
+            featurePrices: # Map<String, BigDecimal> = mapOf(),
+              type: object
+              nullable: true
+              additionalProperties:
+                type: number
+            val featureLimits: # Map<String, Int> = mapOf(),
+              type: object
+              nullable: true
+              additionalProperties:
+                type: integer
+            val sharedFeatureLimits: # Map<String, Int> = mapOf(),
+              type: object
+              nullable: true
+              additionalProperties:
+                type: integer
```

Comment:
1. I think we need to remove `val`s.
2. Is there an OpenAPI construct for maps? If not, these seem fine.

Reply:
Oh, nevermind, looks like you're doing it correctly with `additionalProperties`, as documented here.

Reply:
Now I'm confused haha... `val` parsed just fine with Swagger editor; I think I may be missing something.

Reply:
> On 2: Oh, nevermind, looks like you're doing it correctly with `additionalProperties`, as documented [here](https://swagger.io/docs/specification/data-models/dictionaries/).

Yep, that's where I got that concept 👍

Reply:
> On 1: Now I'm confused haha... `val` parsed just fine with Swagger editor; I think I may be missing something.

Yeah, I don't know why that worked but it did generated a valid SDK method that I have used. But it is weird and inconsistent so I'll move it, and the comments. Obviously I pasted in the definitions from the kotlin file and used them as the template haha.